exonum-java-binding-fakes/pom.xml

```diff
@@ -77,7 +77,7 @@
         <groupId>com.google.code.gson</groupId>
         <artifactId>gson</artifactId>
         <version>2.8.5</version>
-        <scope>compile</scope>
+        <scope>test</scope>
```

Comment:
It is not correct — it is required in the compile scope (which is the default).

Reply:
well, could you please then guide me here what exactly should I do since I'm not so experienced with maven?

Reply:
Just remove the scope because it is the default = compile.